src/ducks/resource.js

```diff
@@ -180,14 +140,21 @@ function selectTranslation(original, translations, type, language) {
 
 function updateTranslation(translation, field, value) {
   return (dispatch) => {
-    const changes = { [field]: value };
+    const changes = { [`strings.${field}`]: value };
     translation.update(changes);
     dispatch({
       type: UPDATE_TRANSLATION,
       payload: translation
     });
-    translation.save()
-    .catch(error => console.error('Update translation error:', error));
+    const { translated_type, translated_id } = translation;
+    translation.save({ translated_type, translated_id })
```

Comment:
this will be the same as the create request passing `translated_*` as query params. 

I'm not sure but does the save function use those function arguments at all?

Reply:
Not as far as I can tell. I added them as an experiment, since this is how you would add params to a GET request with this client.

Reply:
`translation` here is the resource that came back from the API with the initial GET request, which has `translated_type` and `translated_id` as properties. So those are always going to be in the payload when the updated version is passed back to Panoptes on `.save()`.

```javascript
apiClient.type('translations').get({translated_type, translated_id})
.then(([translation]) => {
 translation.update(someChanges).save();
});
```

Reply:
good point - they are serialized attributes for the resource. I had forgotten that and was using them in a different context, apologies. OOI does the update payload contain the diff or the whole updated resource? You shouldn't be able to update the translated type/ id of an existing translation (long term just delete). 

I think the js client does partial PUTs on the diff from memory, API allows it as it was in the spec when we implemented it (though not now https://github.com/json-api/json-api/pull/408)

Reply:
I'm only seeing `translations: { strings }` in the payload for a test update, so I guess it does only send back the properties that have changed.

Interesting that the spec has changed to block that.

Reply:
yeah - it was contentions as PUT is mean to be idempotent, PATCH is now the preferred partial update and we didn't support it for some reason...i can't remember. This is taking me back through a lot of strange decisions...